pkg/pillar/types/zedroutertypes.go

```diff
@@ -22,6 +21,28 @@ import (
 	"github.com/vishvananda/netlink"
 )
 
+// IPAddrNotAvail is returned when there is no (suitable) IP address
+// assigned to a given interface.
+type IPAddrNotAvail struct {
+	IfName string
+}
+
+// Error message.
+func (e *IPAddrNotAvail) Error() string {
+	return "no suitable IP address available"
```

Comment:
See below about including the IfName in the string. Don't know what is common error practice here.

Reply:
Added interface name. Looking at errors from the golang's net package, it seems that common practice is to output all/most of the attributes. Sometimes, through the process of error wrapping, we may get an error message with duplicate information (like interface name mentioned three times), but I guess it is better to have duplicate information than to not have that information at all.